doc/ChangeLog.md

```diff
@@ -22,6 +22,8 @@ All notable changes to the project are documented in this file.
 - Update documentation for use of VETH pairs in containers
 - Issue #454: create bridges in `factory-config` with IGMP/MLD snooping
   enabled by default
+- Add support for optionally running user scripts from
+  `/cfg/user-scripts.d`
```

Comment:
I liked the old name better, `/cfg/start.d` was short and sweet and also illustrative of when the scripts run (at startup-up).

Propose changing to that.

Reply:
Yeah I agree. I was trying to find a common name that would work both for the directory name and the service name.

Should we also change the name of the service in that case? Suggestions?

Reply:
Hmm, good question.  Maybe just plain _start scripts_?  

Considering the Finit [runparts](https://github.com/troglobit/finit/tree/master/doc#run-parts-scripts) supports SXXfoo and KXXbar style notation (appends `start` and `stop` to the script), maybe we want to add support for _stop scripts_ later, i.e., scripts that run at shutdown/reboot?